cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/lineage/field/FieldLineageInfo.java

```diff
@@ -282,7 +283,7 @@ private long computeChecksum() {
     for (WriteOperation write : writeOperations) {
       List<InputField> inputs = write.getInputs();
       for (InputField input : inputs) {
-        Set<String> visitedOperationNames = new HashSet<>();
+        Set<String> visitedOperationNames = new HashSet<>(Collections.singletonList(write.getName()));
```

Comment:
why create a singleton list for this? Better to call ```visitedOperationNames.add(write.getName())```;

Reply:
done